src/Tasks/Query.js

```diff
@@ -126,15 +126,15 @@ export var Query = Task.extend({
     this._cleanParams();
 
     // services hosted on ArcGIS Online and ArcGIS Server 10.3.1+ support requesting geojson directly
-    if (this.options.isModern || isArcgisOnline(this.options.url)) {
+    if (this.options.isModern || (isArcgisOnline(this.options.url) && this.options.isModern === undefined)) {
```

Comment:
@patrickarlt what if someone sets `isModern` to `false`? Should we check that here too or are you intentionally only targeting the case when the user does not set it?

Reply:
If you check also if `isModern === false`, you are not letting the user ask for JSON instead of geoJSON if he wants to ([cf. changelog](https://github.com/Esri/esri-leaflet/blob/master/CHANGELOG.md#added-5))
(That's the reason I found behind this)

Reply:
the `isModern === false` will get caught in the first part of the OR clause, oops - missed that.